dev/tasks/conda-recipes/arrow-cpp/build.sh

```diff
@@ -19,12 +19,14 @@ cmake \
     -DCMAKE_INSTALL_LIBDIR=$PREFIX/lib \
     -DLLVM_TOOLS_BINARY_DIR=$PREFIX/bin \
     -DARROW_DEPENDENCY_SOURCE=SYSTEM \
+    -Duriparser_SOURCE=BUNDLED \
```

Comment:
Note that we need to package `uriparser` before we can release this on conda-forge; there bundling of thirdparties is not really accepted.

Reply:
Thanks for the update, didn't know that.

Reply:
Even for small private dependencies?

Reply:
I think we can temporarily statically-link uriparser, we had a bunch of stuff statically linked on conda-forge for a long time before the CMake refactor